src/main/java/org/jpeek/graph/XmlGraph.java

```diff
@@ -91,16 +88,40 @@ public XmlGraph(final Skeleton skeleton) throws IOException {
             byxml.values()
         );
         for (final XML method : byxml.keySet()) {
-            final List<String> calls = method.xpath("ops/op[@code='call']/text()");
+            final List<XML> calls = method.nodes("ops/op[@code='call']");
             final Node caller = byxml.get(method);
-            for (final String call : calls) {
-                if (byname.containsKey(call)) {
-                    final Node callee = byname.get(call);
+            for (final XML call : calls) {
+                final String name = XmlGraph.call(call);
+                if (byname.containsKey(name)) {
+                    final Node callee = byname.get(name);
                     caller.connections().add(callee);
                     callee.connections().add(caller);
                 }
             }
         }
         return new ListOf<>(byxml.values());
     }
+
+    /**
+     * Serialize method call to a string.
```

Comment:
@HDouss I think that these two private methods could be extracted into two new classes

Reply:
@paulodamaso I left a new puzzle for that.